Automattic/newspack-plugin
# Handle above header placement in Campaigns Wizard

### All Submissions:






### Changes proposed in this Pull Request:



https://github.com/Automattic/newspack-popups/pull/292 introduces an "Above site header" placement option. This PR makes Campaigns Wizard aware of that change.

### How to test the changes in this Pull Request:

1. Follow testing steps in https://github.com/Automattic/newspack-popups/pull/292
2. Visit the Campaigns Wizard, observe that the "above header" campaign is listed as an inline campaign, with a "Above header" note in the card description
3. The card should act as other inline campaigns cards

### Other information:
